tests/cmd/e2e/main.go

```diff
@@ -103,6 +102,46 @@ func main() {
 				BatchSize:   1,
 				RawSize:     1,
 			},
+			SubValues: `pd:
```

Comment:
For better readability, can we move this string literal to a private const value?